src/Features/LanguageServer/Protocol/Handler/RequestContext.cs

```diff
@@ -59,6 +60,10 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 
         public readonly IGlobalOptionService GlobalOptions;
 
+        public readonly LspWorkspaceManager LspWorkspaceManager;
```

Comment:
Tracking issue so we don't have to do this - https://github.com/dotnet/roslyn/issues/61002